paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -253,21 +260,27 @@ def report_waiting_aborted(service, deploy_group):
     paasta_print()
 
 
-def get_authors_to_be_notified(git_url, from_sha, to_sha):
+def get_authors_to_be_notified(git_url, from_sha, to_sha, additional_authors):
     if from_sha is None:
         return ""
-    ret, authors = remote_git.get_authors(
+    ret, git_authors = remote_git.get_authors(
         git_url=git_url, from_sha=from_sha, to_sha=to_sha
     )
-    if ret == 0:
-        if authors == "":
-            return ""
-        else:
-            slacky_authors = ", ".join({f"<@{a}>" for a in authors.split()})
-            log.debug(f"Authors: {slacky_authors}")
-            return f"^ {slacky_authors}"
+    if ret == 1: 
```

Comment:
we should probably add some comments here about why it's fine for this to fail/that we expect this to start failing and additionally create an internal ticket to clean up this logic once we no longer hit any git hosts that call `authors-of-changeset`

Reply:
Added a comment!

Reply:
I think this is fine. What do you think about just making it so we are only using one *or* the other?

And only if author is not provided on the cli do we try to fetch it.

Reply:
@solarkennedy I like that a lot!

Writing it that way feels more useful to a developer triggering the deployment manually. Allowing a dev manually deploying to override who is notified for a deployment is a nice quality of life. 

Additionally, this allows us to continue treating a non-zero return from `remote_git.get_authors` as a failure, instead of swallowing it in some cases. 

I did a quick [internal poll](https://yelp.slack.com/archives/CMSDQP0K1/p1575934938401500)on preferring override vs append, and the result was well in favor of append. 

I've updated the PR with that change.

Reply:
Seems pretty reasonable to me for a cli argument. "tell, don't ask" kinda thing. (we are telling it which authors to use, idk)